### ISSUE TITLE
bug with todos example - clearing all and reloading page triggers debugger

### ISSUE THREAD

**Issue by zangster300** [starfederation/datastar#253]
```
Error evaluating Datastar expression:
  return {'loading ml-4': ctx.store().toggleAllFetching.value}

Error: Cannot read properties of undefined (reading 'value')

Check if the expression is valid before raising an issue.
```

Steps to reproduce:

Toggle all todos
Clear them all at once and refresh

To fix:
Clear cookies for the datastar site

**Comment by delaneyj** [comment]
gotta test for that commited?

**Comment by delaneyj** [comment]
@zangster300  :cricket:

**Comment by delaneyj** [comment]
Fixed in #308
